src/api/spec/controllers/webui/users/rss_tokens_controller_spec.rb

```diff
@@ -4,18 +4,20 @@
   describe 'POST #create' do
     let(:user) { create(:confirmed_user) }
 
-    before do
-      login(user)
+    it_behaves_like 'require logged in user' do
+      let(:method) { :post }
+      let(:action) { :create }
     end
 
     context 'with a user with an existent token' do
-      let(:last_token) { user.create_rss_token.string }
+      let!(:last_token) { user.create_rss_token.string }
```

Comment:
I think there was a little bug in the spec, I've updated the `let!` and the flash message expectation.